CONTRIBUTING.rst

```diff
@@ -0,0 +1,206 @@
+First off, thanks for taking the time to contribute!
+
+The following is a set of guidelines for contributing to Red Hat
+OpenShift PSAP ``ci-artifacts``. These are mostly guidelines, not
+rules. Use your best judgment, and feel free to propose changes to
+this document in a pull request.
+
+---
+
+The primary goal of the repository is to host the tools required for
+the nightly testing of the OpenShift operators under Red Hat PSAP team
+responsibility, and in particular, NVIDIA GPU Operator and the Special
+Resource Operator (SRO).
+
+The OpenShift version we are supporting is 4.N, 4.N-1 and 4.N-2, where 4.N is
+the current latest version released. So as of May 2021, we need to
+support 4.7, 4.6 and 4.5.
+
+The secondary goal of the repository is to offer a toolbox for
+interacting with our operators, and configuring the cluster as required.
+
+Pull Request Guidelines
+-----------------------
+
+- Pull Requests (PRs) need to be ``/approve`` and reviewed ``/lgtm`` by
+  PSAP team members before being merged.
+
+- PRs should have a proper description explaining the problem is being
+  solved, or the new feature being introduced.
+
+- PRs introducing or modifying ``toolbox`` commands should include a
+  documentation commit, so that ``docs`` is kept up-to-date.
```

Comment:
```suggestion
- PRs introducing or modifying ``toolbox`` commands should include a
  documentation commit, so that ``docs`` are kept up-to-date.
```